modules/admin-ui-frontend/app/scripts/modules/events/controllers/eventsController.js

```diff
@@ -49,6 +51,12 @@ angular.module('adminNg.controllers')
       if (!$scope.editorUrl) {
         $scope.editorUrl = '#!/events/events/$id/tools/editor';
       }
+      if (angular.isDefined(user.org.properties['adminui.embed_button_display'])) {
+        var embedButtonConfig = user.org.properties['adminui.embed_button_display'];
+        if (embedButtonConfig === 'false') {
+          $scope.showEmbedButtons = embedButtonConfig;
+        }
```

Comment:
You set `showEmbedButtons` to the boolean value `true` or the string `"false"` which may work (not actually sure) but is confusing and may break eventually. You could use something like this to always set this to a boolean value:

```suggestion
        $scope.showEmbedButtons = user.org.properties['adminui.embed_button_display'] !== 'false';
```

Reply:
Makes sense, updated the code with your suggested change.